MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Layout in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the Layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Layout in .NET MAUI DataForm (SfDataForm)
+
+## Overview
+
+The data form supports linear and grid layouts and manages layout of label, editor, and validation label.
+
+## Linear data form layout
+
+By default, the data form arranges the fields one-by-one. It is applicable for both label positions: left and top.
+
+When the label position is Left, the linear layout is shown as follows:
+
+When the label position is Top, the linear layout is shown as follows:
+
+## Grid data form layout
+
+By default, the data form arranges one data field per row. It is possible to have more than one data fields per row by setting the [ColumnCount] property which provides grid like layout for the data form. The column order of data form field in a row can also be changed using [ItemsOrderInRow].
```

Comment:
By default, the data form arranges one data field per row. It is possible to have more than one data field per row by setting the [ColumnCount] property which provides a grid-like layout for the data form. The column order of the data form field in a row can be changed using the [ItemsOrderInRow].

Reply:
Content changed